transform/snowflake-dbt/models/data_quality_layer/raw/rudderstack/dq_daily_activity.sql

```diff
@@ -0,0 +1,63 @@
+{{config({
+    "materialized": 'table',
+    "schema": "data_quality",
+    "tags":'data-quality'
```

Comment:
Are we creating a new dbt cloud job that runs tag data-quality?
or we can also add it to dbt-nightly if its meant to run once per day.